lib/Arch/Sleigh/X86Arch.cpp

```diff
@@ -32,22 +32,49 @@
 namespace remill {
 namespace sleigh::x86 {
 
-class SleighX86Arch final : public SleighArch, public remill::X86ArchBase {
+class SleighX86Decoder final : public SleighDecoder {
+ public:
+  SleighX86Decoder() = delete;
+  SleighX86Decoder(const remill::Arch &arch)
+      : SleighDecoder(
+            arch, kArchX86_SLEIGH == arch.arch_name ? "x86.sla" : "x86-64.sla",
+            kArchX86_SLEIGH == arch.arch_name ? "x86.pspec" : "x86-64.pspec",
+            {}) {}
+
+  virtual ~SleighX86Decoder(void) = default;
+
+  void InitializeSleighContext(
```

Comment:
Add comment why it is not ok to just use the parent impl but instead an empty body is specifically defined (at least from the PR I did not deduce it in reasonable time).